packages/ic-sha3/BUILD.bazel

```diff
@@ -0,0 +1,47 @@
+load("@rules_rust//rust:defs.bzl", "rust_doc", "rust_doc_test", "rust_library", "rust_test", "rust_test_suite")
+
+package(default_visibility = ["//visibility:public"])
+
+DEPENDENCIES = [
+    # Keep sorted.
+    "@crate_index//:sha3",
+]
+
+DEV_DEPENDENCIES = [
+    # Keep sorted.
+    "@crate_index//:hex",
+]
+
+rust_library(
+    name = "ic-sha3",
+    srcs = glob(["src/**/*.rs"]),
+    deps = DEPENDENCIES,
+)
+
+rust_test(
+    name = "unit_tests",
+    crate = ":ic-sha3",
+)
+
+rust_test_suite(
+    name = "integration_tests",
```

Comment:
We sometimes skip the "_tests" part for the `rust_test_suite`, because it is implicitly added by Bazel:
```
/ic$ bazel query //packages/ic-sha3/...
[...]
//packages/ic-sha3:integration_tests
//packages/ic-sha3:integration_tests_tests/keccak256
//packages/ic-sha3:integration_tests_tests/shake256
//packages/ic-sha3:unit_tests
```

However, it is nicer to call `bazel test //packages/ic-sha3:integration_tests` rater than `bazel test //packages/ic-sha3:integration` to run the entire suite, so I'd say we leave this as is.

Reply:
yes I also saw that, but I thought I would leave as is to preserve the current behaviour.